svm/tests/integration_test.rs

```diff
@@ -2288,13 +2358,109 @@ fn simd83_account_reallocate(enable_fee_only_transactions: bool) -> Vec<SvmTestE
 #[test_case(simd83_fee_payer_deallocate(true))]
 #[test_case(simd83_account_reallocate(false))]
 #[test_case(simd83_account_reallocate(true))]
+#[test_case(program_cache_update_tombstone())]
 fn svm_integration(test_entries: Vec<SvmTestEntry>) {
     for test_entry in test_entries {
         let env = SvmTestEnvironment::create(test_entry);
         env.execute();
     }
 }
 
+#[test_matrix([false, true], [false, true])]
+fn program_cache_create_account(enable_fee_only_transactions: bool, disable_executable_flag: bool) {
```

Comment:
disable_executable_flag => remove_accounts_executable_flag_checks

It only removes the checks of the flag, not the flag itself.